### PR TITLE
simplify docker-compose traefik config

### PR DESCRIPTION
**Pull request by ForsakenHarmony**
All done in the web editor, so not tested 🙃

But the traefik tweaks are taken from my server

### CODE REVIEW

**Comment by pizkaz** [comment]
Oops, totally over-looked this PR. @tabascoeye, could you check docker-compose.prod.singledomain.yaml?

File: back/Dockerfile [hunk 1]
```diff
@@ -23,4 +23,5 @@ ENV NODE_ENV=production
 RUN yarn install --production
 
 USER node
+EXPOSE 8080
```
Comment (pizkaz) [inline back/Dockerfile:26]: Hmm refresh my memory plz: These `EXPOSE` directives are purely for documentation, right?

Comment (tabascoeye) [reply back/Dockerfile:26]: "The EXPOSE instruction does not actually publish the port. It functions as a type of documentation between the person who builds the image and the person who runs the container, about which ports are intended to be published. To actually publish the port when running the container, use the -p flag on docker run to publish and map one or more ports, or the -P flag to publish all exposed ports and map them to high-order ports."
looks like it

**Comment by tabascoeye** [comment]
Does NOT work for me unfortunately. ACME is not able to get a certificate because the domain names are not as expected:

````
reverse-proxy_1  | time="2021-12-07T13:37:27Z" level=error msg="Unable to obtain ACME certificate for domains \"reverse-proxy-docker\": unable to generate a certificate for the domains [reverse-proxy-docker]: acme: error: 400 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rejectedIdentifier :: Error creating new order :: Cannot issue for \"reverse-proxy-docker\": Domain name needs at least one dot, url: " providerName=myresolver.acme routerName=websecure-reverse-proxy-docker@docker rule="Host(`reverse-proxy-docker`)"
````

**Comment by ForsakenHarmony** [comment]
> Does NOT work for me unfortunately. ACME is not able to get a certificate because the domain names are not as expected:

That's interesting, I guess the fix would be to remove exposed by default and enable for each container

Does that only happen for the reverse proxy or the other ones as well?

**Comment by tabascoeye** [comment]
looks like only the reverse proxy

**Comment by pizkaz** [comment]
Any updates on this? It's been quite a long time. If not: Are you fine with closing this PR?

**Pull request closed without merging**